mono/metadata/environment.c

```diff
@@ -70,6 +70,8 @@ ves_icall_System_Environment_GetOSVersionString (MonoError *error)
 #elif defined(HAVE_SYS_UTSNAME_H)
 	struct utsname name;
 
+    memset (&name, 0, sizeof (name)); // WSL does not always nul terminate.
```

Comment:
Indented wrong, will fix.

Reply:
indent is still wrong, can you fix that?